recipes/vorbis/all/conanfile.py

```diff
@@ -34,7 +34,7 @@ def config_options(self):
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
```

Comment:
I think that we could add the `self.settings.rm_safe("xxx")` as well (lines below ⬇️ )